lib/exportPlatform.js

```diff
@@ -41,9 +44,16 @@ function exportPlatform(platform) {
   // values like "1px solid {color.border.base}" we want to
   // transform the original value (color.border.base) before
   // replacing that value in the string.
-  return resolveObject(
+  to_ret = resolveObject(
     transformObject(this.properties, platformConfig)
   );
+
+  if(handleErrors.count(ERROR_TYPE)) {
```

Comment:
Do we need to check for the type of error? If there are other errors wouldn't we want to show them anyways?

Reply:
The intent of this code is to provide a grouping mechanism for errors. If there are other errors, they will be thrown immediately in the code executed much more deeply. If we make it out to this point, it suggests that we didn't have other errors so we should take our collected errors and display them all at once and then throw.

I could see how this looks like how we should handle errors in general. If thats the case I should rework ALL other times we spit out errors. Probably needs to be resolved in relationship to the options.log = error conversation as well.